src/main/java/com/notably/logic/suggestion/handler/SearchSuggestionArgHandler.java

```diff
@@ -45,6 +51,7 @@ public SearchSuggestionArgHandler(Model model) {
         }
 
         if (keyword.isEmpty()) {
+            logger.warning("keyword is empty");
```

Comment:
This shouldn't be a warning? Since it's a suggestion and it's not really an invalid thing to have a blank keyword